### PR TITLE
Gui testing

### PR DESCRIPTION
**Pull request by jacobRidenour**
* Update GUI to use bootstrap
   * Make it so MIDI output/PDF output disappear when a new file is selected
   * Fix bug: Clicking Get PDF multiple times adds new PDF to the right of the previous (unintended behavior)
* Add temp file cleanup for basic-pitch MIDI predictions (default is 30 files, feel free to change)
* Add/update requirements documentation in /background
* Add test audio files test.mp3 and test2.mp3 for reference
* Trained new model (update notebook, instrument recognizer)
   * Update requirements.txt to include additional libraries for the model notebook
   * Add model .keras -> .h5 conversion at the end of model notebook
   * Use stereo audio instead of mono
      * Update instrument_recognizer.py to work with stereo audio
   * Add data augmentation (pitch shift, reverse channels, add noise) - accuracy improved by ~23% but still overfitting a bit
   * Add template document in /instrument_recognition/Observations.md so we can easily compare our structure/results as we change things

### CODE REVIEW

File: test.mp3
Comment (TreytonPerkins) [inline test.mp3:1]: Should we add the mp3 files (or a folder of test files) to the gitignore?

Comment (jacobRidenour) [reply test.mp3:1]: Can do, I was thinking it would be useful to have a consistent audio file between us all for testing. What do you guys think?

Comment (AustinShutt) [reply test.mp3:1]: Yeah, we could probably put them in some sort of file storage on the shared drive?

**Review by AustinShutt — APPROVED** [review]
Looks good! I'm excited to play around with it. Looks like we're doing a lot better with the overfitting problem through data augmentation. I'm curious how the changes to the model structure I was playing with will coincide with that. 

Great work!

File: test.mp3
Comment (jacobRidenour) [reply test.mp3:1]: Done!

**Review by TreytonPerkins — APPROVED** [review]
